dev-requirements.txt

```diff
@@ -21,3 +21,4 @@ six==1.10.0
 snowballstemmer==1.2.1
 Sphinx==1.4.8
 sphinx_rtd_theme==0.1.9
+imbalanced-learn==0.2.1
```

Comment:
I'm guessing this will be used in a subsequent PR (and not here). Wanted to make sure I didn't miss its use...